services.md

```diff
@@ -0,0 +1,101 @@
+
+> Note: this setup does not include a site redesign.
+
+**Goals:** Gets your project a web presence as quickly as possible.
+
+**What we need from you:** A github repository, preferably named after the website domain name (i.e. longhorn/longhorn.io, rather than longhorn/website)
+Netlify installed/added to this repository
+A list of key maintainer(s) who should have access to Netlify
+A decision on whether your project will use CNCF’s CLA or Github’s DCO for committer verification. 
+
+
+## Techdocs office hours
+
+**What it is:** An hour long open meeting with CNCF’s technical writers where projects can ask for questions and advice on their documentation.
+
+**Goals:** A way for non-documentation experts to access documentation experts as needed. 
+
+>Note: Office hours works best for smaller, tactical questions ("I need to write a getting started guide. Where do I start?") For more general questions, we recommend requesting a docs assessment from the team first.
+
+**What we need from you:** Show up with a well-scoped question!
+
+
+## Website branding & design work
+
+**What it is:** We will do (or contract out) a redesign of your website and add any features you feel you want and can maintain independently, in order for your project to present itself professionally. 
+
+**Goals:** A way for projects that don’t have a supporting organization’s design/web dev departments to have a professional-feeling website, increasing project trust.
+
+**What we need from you:** A demonstration of need that is scoped enough to draft a statement of work for a contractor, if required.
+
+> Note: We are a small team and these projects are time intensive. As such, we can only offer a few slots per year for these kinds of projects. 
+
+## Well-scoped writing projects
+
+**What it is:** Have a CNCF technical writer work on a well-scoped, small-to-medium sized writing project that is accomplishable within 1-2 2 week sprints.
+
+Examples of well-scoped projects:
+
+- Writing individual tutorial(s), feature documentation, or a getting started guide from scratch
+- Intensive editing for language/grammar of one section of your documentation
+- Review of your information architecture (documentation organization/structure) and proposing improvements
+
+
+Examples of poorly scoped projects:
+
+- Asking for help to “Improve” a section of documentation 
+- “Write tutorials”
+
+**Prerequisites:** A completed documentation assessment which outlines the need for a specific kind of documentation that does not exist already.
+
+**Goals:** Improve your documentation in measurable ways that you may not have the resources to accomplish. 
+
+**What we need from you:** A ‘point person’ or maintainer knowledgeable in the area they’re being asked to work on to ask questions to be available 1-5 hours/week for the duration of the project. 
+
+
+## Contractor funding 
```

Comment:
I'd remove this section on contractor funding as this is more of a "how" we go about completing a specific service, whether it's a short term or long term scoped engagement is what I'm really looking for, those engagements can be completed in house or hiring contract work to help